### PR TITLE
GROK-12428: Bio | Tools | Dendrogram: Tree colors

### PR DESCRIPTION
**Pull request by tanas80**
Harmonize line/stroke width, node size, line/stroke color, and fill color properties for Dendrogram, PhylocanvasGL, and PhyloTree viewers. And use standard colors from bio lib as defaults.

### CODE REVIEW

File: packages/PhyloTreeViewer/src/viewers/phylocanvas-gl-viewer.ts [hunk 1]
```diff
@@ -32,44 +68,47 @@ const TreeTypesTypes: { [treeType: string]: string } = {
   [TreeTypesNames.Orthogonal]: TreeTypes.Hierarchical, // rectangular edges, leaves listed horizontally
 };
 
+function setProps(value: any) {
+
+}
```
Comment (StLeonidas) [inline packages/PhyloTreeViewer/src/viewers/phylocanvas-gl-viewer.ts:73]: empty function?

File: libraries/utils/src/color.ts [hunk 2]
```diff
@@ -11,6 +11,31 @@ export function setAlpha(color: DG.ColorType, alpha: number): number {
...
+}
+
+/** Converts color Hex string ('#000bff') to int number */
+export function htmlToInt(value: string): number {
+  return DG.Color.fromHtml(value) & 0xFFFFFF;
+}
+
+/** Converts color Hex string ('#000bff22' , alpha=0x22) to int number with alpha */
+export function htmlToIntA(value: string): number {
+  const m: RegExpExecArray = /#([0-9A-Fa-f]{6})([0-9A-Fa-f]{2})/.exec(value)!;
```
Comment (StLeonidas) [inline libraries/utils/src/color.ts:39]: comment regular expression

Comment (tanas80) [reply libraries/utils/src/color.ts:39]: /* regexp: dies sign + six hex digits (group 1 for rgb) + two hex digits (group 2 for alpha) */

File: packages/PhyloTreeViewer/src/viewers/phylocanvas-gl-viewer.ts (hunk 1 above)
Comment (tanas80) [reply packages/PhyloTreeViewer/src/viewers/phylocanvas-gl-viewer.ts:73]: not used, removed

**Comment by tanas80** [comment]
Сode review comments taken into account

**Pull request merged**